src/v/cloud_io/tests/s3_imposter.cc

```diff
@@ -336,8 +336,7 @@ struct s3_imposter_fixture::content_handler {
               fixt_log.trace, "S3 imposter response: {}", repl.response_line());
             return "";
         } else if (
-          request._method == "POST"
-          && request.query_parameters.contains("delete")) {
+          request._method == "POST" && request.has_query_param("delete")) {
```

Comment:
method() next

Reply:
Next they will introduce a `method()` function such that one doesn't have to access the underscore member anymore.